### PR TITLE
add people to current page

### PR DESCRIPTION
**Pull request by nelsonpecora**
* allows adding other people to the current page
* allows you to search through directory of people, to find the best person
* this does not notify the person added, but does put the page in their "My Pages" list

### CODE REVIEW

File: panes/add-person-to-page.vue [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+          this.users = users;
+        });
+      },
+      onSearchKeyup: _.debounce(function () {
+        this.fetchUsers();
+      }, 300),
+      addPersonToPage(user) {
+        this.$store.dispatch('startProgress', 'save');
+        return this.$store.dispatch('updatePageList', { user }).then(() => {
+          this.$store.dispatch('finishProgress', 'save');
```
Comment (mheiber) [inline panes/add-person-to-page.vue]: I'd recommend adding `finishProgress` and closing the pane and such to the `catch` for the promise as well, unless the error handling is already happening elsewhere

Comment (nelsonpecora) [reply panes/add-person-to-page.vue]: Good point, I need to catch that if it fails for some reason

File: panes/add-person-to-page.vue [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+    </div>
+  </div>
+</template>
+
+<script>
+  import _ from 'lodash';
+  import { postJSON } from '../lib/core-data/api';
+  import { searchRoute } from '../lib/utils/references';
+
+  function buildUserQuery(query) {
```
Comment (mheiber) [inline panes/add-person-to-page.vue]: nit:
buildUserQuery isn't a UI thing—by pulling it out of the Vue file it would be easier to test

**Comment by mheiber** [comment]
code LGTM modulo 2 nits. Want me to run it and test in a browser?

File: panes/add-person-to-page.vue (hunk 2 above)
Comment (nelsonpecora) [reply panes/add-person-to-page.vue]: Eh, the logic for it is super simple. There's not much to test 😅

**Pull request merged**